api/v1alpha1/jwt_types.go

```diff
@@ -108,6 +110,12 @@ type RemoteJWKS struct {
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=253
 	URI string `json:"uri"`
+	// Duration after which the cached JWKS should be expired. If not specified, default cache duration is 5 minutes.
+
+	// +kubebuilder:default="300s"
+	// +kubebuilder:validation:Format=duration
```

Comment:
lets rm `// +kubebuilder:validation:Format=duration`, we are relying on CEL validations of `gwapiv1.Duration`